src/elements/service/ServiceList.jsx

```diff
@@ -26,11 +26,12 @@ const ServiceList = [
         icon: <FiUsers />,
         title: 'Public Speaking or Pod cast requests',
         description: 'Check my Social Media for updates.'
-    },
+    }
+    // ,
     // { 
     //     icon: <FiMonitor />,
-    //     title: 'Mobile App Development',
-    //     description: 'I throw myself down among the tall grass by the stream as I lie close to the earth.'
+    //     title: '',
+    //     description: ''

```

Comment:
It's always a good idea to remove comments like these when we are not using the code.

I like to use comments to help explain what is going on with the code. But you have to keep in mind that your code already tells a story; so you might not need a comment. Most of the time you are not going to need one.

For example:
```ts
  // this creates a dog <-- does not add new information
  function createDog() {
    return new Dog();
  }
```

Reply:
Sorry, I forgot to add that I think we should come back to this one later.

Let's bookmark this with a new GitHub issue for now. 🙂